src/base/task.cc

```diff
@@ -851,7 +855,7 @@ void TaskScheduler::WaitForTerminateCompletion() {
 
     int count = 0;
     int threadsRunning;
-    pid_t pid = getpid();
+    pid_t pid = osspecific_getpid();
```

Comment:
yes, I agree with the comment. 
But current build  is expected to fail for linux. In fact, it will not compile for linux at all even if several such issues are fixed.
We will add #ifdefs when we merge with the main branch in Contrail as per our discussion.